postgres/schemas/models.py

```diff
@@ -85,7 +86,7 @@ class Keywords(Base):
     channel_program = Column(String, nullable=True) #  arcom - alembic handles this
     channel_program_type = Column(String, nullable=True) # arcom - (magazine, journal etc) alembic handles this
     channel_radio = Column(Boolean, nullable=True)
-    start = Column(DateTime())
+    start = Column(DateTime(), primary_key=True)
```

Comment:
missing from production, it should have been added before (on partition creation)